RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -294,6 +294,72 @@ boolean LX_getStarKey()
 
 boolean beehiveConsider()
 {
+	int damage_sources = 1; // basic hit
```

Comment:
could you share code with the relevant bit in `level_13.ash`?

Reply:
also the preparatory function in `level_13.ash` doesn't even ensure prismatic damage is present (as far as I can tell) -- it just uses whatever you already have.